datahub-web-react/src/app/previewV2/DefaultPreviewCard.tsx

```diff
@@ -117,6 +127,7 @@ const ENTITY_TYPES_WITH_DESCRIPTION_PREVIEW = new Set([
     EntityType.DataProduct,
     EntityType.Domain,
     EntityType.Tag,
+    EntityType.Dataset,
```

Comment:
if this flag is enabled I think we should try to render description for any entity type right? like charts, dashboards, containers, etc. as well